roles/openshift_logging_fluentd/tasks/main.yaml

```diff
@@ -173,6 +172,7 @@
     ops_port: "{{ openshift_logging_fluentd_ops_port }}"
     fluentd_nodeselector_key: "{{ openshift_logging_fluentd_nodeselector.keys()[0] }}"
     fluentd_nodeselector_value: "{{ openshift_logging_fluentd_nodeselector.values()[0] }}"
+    audit_container_engine: "{{ openshift_logging_audit_container_engine }}"
```

Comment:
In `roles/openshift_logging_fluentd/defaults/main.yaml` you should define a default value:
```
openshift_logging_audit_container_engine: False
```

Reply:
I would suggest that if its going to live in the `openshift_logging_fluentd` role we preface it with that role's name.